avro/src/writer.rs

```diff
@@ -658,6 +656,19 @@ pub fn to_avro_datum<T: Into<Value>>(schema: &Schema, value: T) -> AvroResult<Ve
     Ok(buffer)
 }
 
+/// Write the referenced `Serialize` object to the provided Write object. Returns a result with
+/// the number of bytes written.
+///
+/// **NOTE** This function has a quite small niche of usage and does NOT generate headers and sync
+/// markers; use [`Writer`](struct.Writer.html) to be fully Avro-compatible if you don't know what
```

Comment:
> use [`Writer`](struct.Writer.html) to be fully Avro-compatible if you don't know what

I don't quite follow you here. What do you mean by `use Writer instead` ?
IMO it would be clearer to say something like `use write_ref(...) instead`.

Reply:
So this was me copying the comment from `to_avro_datum`.

The issue with this and that function is they look like what someone would want, they're simple to call but they will do the wrong thing for most users.

I think the idea was to direct people to the Writer struct documentation which is what I imagine most people would want? I can change the link to (ignore the off formatting, can't write markdown in markdown)
['append_ser'] (struct.Writer.html#method.append_ser)